src/building/prerequisites.md

```diff
@@ -0,0 +1,29 @@
+# Prerequisites
+
+Before building the compiler, you need the following things installed:
+
+* Python
+* A C/C++ compiler toolchain
+* cmake
+* rustc
+
+## `rustc` and toolchain installation
+
+Follow the installation given in the [Rust book](https://doc.rust-lang.org/book/ch01-01-installation.html) to install a working `rustc` and the necessary C/++ toolchain on your platform.
+
+## Platform specific instructions
```

Comment:
On the linux side, you can install `llvm-N` and `llvm-N-tools` installed if you want to build with system LLVM (rather than building LLVM from scratch, which takes a long time).

I'm not aware that anything else is needed, and I don't think you need `cmake` on linux either.

Reply:
You need cmake if you want to compile LLVM from source AFAIK.